roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -1,4 +1,8 @@
 ---
+- fail:
+    msg: "use_manageiq requires OpenShift Enterprise 3.1 or Origin 1.1."
+  when: openshift.common.version_greater_than_3_1_or_1_1 | bool == False
```

Comment:
I'd probably just go with:
```
when: not openshift.common.version_greater_than_3_1_or_1_1 | bool
```
but that is just being pedantic :)

Outside of that, I would update the message to be "The openshift_manageiq role requires..."